FreshRSS/FreshRSS
# Fix: subscription management: mark empty category

Before:
![grafik](https://user-images.githubusercontent.com/1645099/137387372-99143f9b-065b-461c-ac29-514c74ca1f71.png)

After:
![grafik](https://user-images.githubusercontent.com/1645099/137387321-98755762-3ba1-410f-9ca8-fe08095355d3.png)


Changes proposed in this pull request:
- if a category is empty, than a warn message box is shown

How to test the feature manually:
1. go to subscription management
2. delete a category
3. see the box

Pull request checklist:
